shaku/tests/simple_generic_component.rs

```diff
@@ -0,0 +1,60 @@
+//! Modules and services can be generic. Based off of issue #2:
+//! https://github.com/Mcat12/shaku/issues/2
+
+use shaku::{module, Component, Interface, HasComponent};
+
+trait Animal {
+   fn audible_identification(&self) -> &str;
```

Comment:
Maybe instead of `identification` use `sound`?

Reply:
Done.